components/MovieCard/index.js

```diff
@@ -19,22 +15,18 @@ export default function MovieCard({
   return (
     <MovieListItem>
       <CardImage
-        onClick={() => {
-          if (isOpen) {
-            setActiveItem(undefined);
-            return;
-          }
-          setActiveItem(index);
-        }}
+        onClick={() =>
+          setIsDetailShown((previousIsDetailShown) => !previousIsDetailShown)
+        }
         src={"https://image.tmdb.org/t/p/w500/" + movie.poster_path}
       ></CardImage>
       <>
-        {isOpen ? (
+        <UserRating>User-Rating: {movie.vote_average}</UserRating>
+        <StyledBookmark onClick={() => onToggleBookmark(movie.id)}>
+          {bookmarks.includes(movie.id) ? "REMOVE" : "ADD"}
+        </StyledBookmark>
+        {isDetailShown ? (
```

Comment:
Instead of using a ternary and returning fragments without content (`: (<></>)` ), I suggest the following:

```suggestion
        {isDetailShown && (<CardBody>...</CardBody>)}
```